runtime/runtime/src/lib.rs

```diff
@@ -1268,6 +1279,19 @@ impl Runtime {
             }
 
             total_gas_burnt = safe_add_gas(total_gas_burnt, outcome_with_id.outcome.gas_burnt)?;
+            total_compute_usage = safe_add_compute(
+                total_compute_usage,
+                outcome_with_id.outcome.compute_usage.unwrap(),
```

Comment:
Can we add a comment why `unwrap` cannot fail here? Or sometimes I also like to use `expect("<insert impossible condition> happened")` instead. Eitherway, someone reading the code should quickly understand that this is an unreachable panic and why.